apps/reference/docs/guides/realtime.mdx

```diff
@@ -1,44 +1,54 @@
 ---
 id: realtime
 title: Realtime
-description: Using Supabase Realtime for multiplayer functionality.
+description: Supabase Realtime with Broadcast, Presence, and Postgres CDC.
 sidebar_label: Overview
 ---
 
-Supabase provides a Realtime API using [Realtime](https://github.com/supabase/realtime).
+Supabase provides a globally distributed cluster of [Realtime](https://github.com/supabase/realtime) servers that enable the following functionality:
 
-You can use this to:
+- [Broadcast](#broadcast): Send ephemeral messages from client to clients with low latency.
+- [Presence](#presence): Track and synchrononize shared state between clients.
+- [Postgres CDC](#postgres-cdc): Listen to Postgres database changes and send them to authorized clients.
 
-- broadcast messages between users
-- manage shared state between users
-- listen to database changes over websockets
+A [channel](https://hexdocs.pm/phoenix/channels.html) is the basic building block of Realtime and narrows the scope of data flow to subscribed clients. You can think of a channel as a chatroom where participants are able to see who's online and send and receive messages; similar to a Discord or Slack channel.
 
-## Realtime Channels
+All clients can connect to a channel and take advantage of the built-in features, Broadcast and Presence, while extenstions, like Postgres CDC, must be enabled prior to use.
 
-In Realtime, everything is a [_channel_](https://hexdocs.pm/phoenix/channels.html).
-Channels are like chatrooms where participants can join and leave—similar to Slack or Discord channels.
+## Features
 
 ### Broadcast
 
-Broadcast follows the classic Publisher/Subscriber pattern. A client (publisher) “broadcasts” messages using a unique identifier. For example, a user could send a message to a `#random` channel.
+Broadcast follows the [publish-subscribe pattern](https://en.wikipedia.org/wiki/Publish%E2%80%93subscribe_pattern) where a client publishes messages to a channel with a unique identifier. For example, a user could send a message to a channel with id `room-1`.
 
-Other clients (subscribers) can “listen” to these messages in real-time, using the unique identifier. If they are listening to the `#random` channel, then they will receive the message.
+Other clients can elect to receive the message in real-time by subscribing to the channel with id `room-1`. If these clients are online and subscribed then they will receive the message.
 
-A common use-case for Broadcast is sharing a user's cursor position with other clients in an online game.
+If you've used Realtime prior to the introduction of Broadcast and Presence, you've actually been using this feature all along. Broadcast is the delivery mechanism that sends Postgres changes to your clients. Now, you can send messages directly from client to other clients in a channel without having to persist them in the database.
```

Comment:
This is probably fine for a blog post but we should omit assumptions like these from the docs. It's "nice-to-know" if they've been using it but likely to just confuse new users.

Reply:
https://github.com/supabase/supabase/pull/9359/commits/157670a3e2474c93660e07005a780da79437b1c1